homeassistant/components/unifi/sensor.py

```diff
@@ -100,3 +125,31 @@ def state(self) -> int:
         if self._is_wired:
             return self.client.wired_tx_bytes / 1000000
         return self.client.tx_bytes / 1000000
+
+
+class UniFiUpTimeSensor(UniFiClient):
+    """UniFi uptime sensor base class."""
+
+    DOMAIN = DOMAIN
+    CLASS = DEVICE_CLASS_TIMESTAMP
```

Comment:
Where is `CLASS` used?

Reply:
The device class needed to be the timestamp class right?

Reply:
We only need to set the `device_class` property correctly which is already done. We can remove this.

Reply:
Please remove this.
```suggestion
```